test/e2e/upgrade_test.go

```
@@ -100,35 +70,19 @@ func testTamperAnnotation(t *testing.T) {
 	testPrometheus(t)
 }
 
-// configureUpgradeTest carries out steps required before running tests for upgrade scenario.
-// The steps include -
-// 1. Scale down the operator to 0.
-// 2. Change Windows node version annotation to an invalid value
-// 3. Scale up the operator to 1
+// configureUpgradeTest carries out steps required before running tests for upgrade scenario. The steps include -
+// 1. Ensuring the correct amount of nodes have a proper annotation to start
+// 2. Ensuring the operator is running with 1 replica
 func (tc *testContext) configureUpgradeTest() error {
-	// Scale down the WMCO deployment to 0
-	if err := tc.scaleWMCODeployment(0); err != nil {
-		return err
-	}
-
-	// Override the Windows Node Version Annotation
-	nodes, err := tc.client.K8s.CoreV1().Nodes().List(context.TODO(), metav1.ListOptions{LabelSelector: nc.WindowsOSLabel})
+	nodes, err := tc.client.K8s.CoreV1().Nodes().List(context.TODO(),
```

Example of option 2, guard + fail fast